src/components/HrViewElements/FilteringRatingSteps.tsx

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+export const FilteringRatingSteps = () => (
+    <div className="w-full flex justify-between text-md px-2">
+        <span>1</span><span>2</span><span>3</span><span>4</span><span>5</span>
```

Comment:
To powinno być zautomatyzowane. Ot, chociażby `{new Array(5).fill(null).map((_, i) => <span>{i+1}</span>)}`